db/batchlog_manager.cc

```diff
@@ -290,7 +290,7 @@ future<> db::batchlog_manager::replay_all_failed_batches() {
             mutation m(schema, key);
             auto now = service::client_state(service::client_state::internal_tag()).get_timestamp();
             m.partition().apply_delete(*schema, clustering_key_prefix::make_empty(), tombstone(now, gc_clock::now()));
-            return _qp.proxy().mutate_locally(m);
+            return _qp.proxy().mutate_locally(m, tracing::trace_state_ptr(nullptr));
```

Comment:
No need to be this verbose, you can just pass `nullptr`, or even just `{}`.

Reply:
explicit > implicit. I did this to avoid this kind of calls (from our source):
```cpp
return mutate_internal(std::move(ms), db::consistency_level::ALL, false, nullptr, empty_service_permit(), timeout);
```
I really have no clue what `true` parameter means just by looking at the call site, even though I checked it today in the morning. Being explicit here does not really cost anything aside from a couple of nanoseconds of preprocessing, while it makes the code more self-explanatory.
If you don't mind, I'd rather it stayed :)

P.S. that's also why I really like the idea of using our `bool_class` instead of plain bools as parameters.

Reply:
Sure, that is aright, I won't object to more readable call-sites.

Reply:
I think using `tracing::trace_state_ptr()` would be cleaner.